src/Authentication/TokenGenerator/JWT/JWTGeneratorInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace CodeIgniter\Shield\Authentication\TokenGenerator\JWT;
+
+interface JWTGeneratorInterface
```

Comment:
Like the DecoderInterface, it doesn't appear that we're setting things up to swap out JWT implementations so I don't think an interface is necessary. Making it possible to swap out implementations seems like an unnecessary complication for very minimal gain.